client/app.jsx

```diff
@@ -31,7 +32,11 @@ export default class App extends React.Component {
     const { route } = this.state;
     if (route.path === 'song-form') {
       return <SongForm></SongForm>;
-    } else if (route.path === '') {
+    } else if (route.path === 'recommendations') {
+      return <Recommendations></Recommendations>
```

Comment:
You can pass the `route.params` to Recommendations as a prop so you don't have to do surgery to reconstruct the params.